packages/api/test/e2e/queries.e2e.ts

```diff
@@ -132,4 +132,24 @@ describe('e2e queries', () => {
        done();
     });
   });
+
+  describe('Staking account derived query', () => {
+    it("Get staking account details", async done => {
+      const stashId = '5GNJqTPyNqANBkUVMN1LPPrxXnFouWXoe2wNSmmEoLctxiZY';
+      const stakingAccount = await api.derive.cennznetStaking.query(stashId);
+      expect(stakingAccount).toBeDefined();
```

Comment:
Could we explicitly display expected results in test cases via `toBe()` or `toEqual()`, once all of the values are not dynamic, so that staking account structure and data type could be understood much easier?

Reply:
Would it be easy to just copy results from `https://app.circleci.com/pipelines/github/cennznet/api.js/412/workflows/d3563690-f72d-43e4-a8bf-818255ed62ca/jobs/414`, and remove `console.log` below?